plugins/brimcap/brimcap-plugin.ts

```diff
@@ -214,11 +217,78 @@ export default class BrimcapPlugin {
     )
   }
 
+  private setupLoader() {
+    const load = async (
+      params: IngestParams & {spaceId: string},
+      onProgressUpdate: (value: number | null) => void,
+      onWarning: (warning: string) => void,
+      onDetailUpdate: () => void
```

Comment:
I think onDetailUpdate should be implied when you call onProgressUpdate.

Reply:
I had originally paired them together but it produced some unwanted re-renders since I couldn't replicate the exact logic we use in https://github.com/brimdata/brim/blob/main/src/js/flows/ingestFiles.ts#L161-L195 without the ability to update details *without* updating progress.